play/PlayerRules/CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt

```diff
@@ -39,36 +39,38 @@ Adrenaline Cost:30
 Needle Cost:	1
 Duration: 	2 Actions
 Range:		40m
-Effect:		10 % chance to miss. Deal 52 piercing damage.
-If target has no shield left and is still alive, apply a 
+Effect:		2 chance to miss. Deal 52 piercing damage. AP level 3.
+If the needle penetrates the targets armor and is still alive, apply a 
 poison payload of your choice.
 
 /////	Fast Spike
-Adrenaline Cost:30
+Adrenaline Cost:35
 Needle Cost:	1
 Duration: 	1 Action
 Range:		40m
-Effect:		16 % chance to miss. Deal 52 piercing damage.
-If target has no shield left and is still alive, apply a 
-poison payload of your choice.
+Effect:		2 chance to miss. Deal 52 piercing damage. AP level 3.
```

Comment:
soooooooo, is "Piercing Damage" flavor text or does it need to be defined?

Reply:
Flavor, really. Think I should yank it?

Reply:
I think yanking it is a good idea. I don't mind it now that I know it's flavor text, but nobody else knows, and several games I've played use 'piercing' as a damage type.

Reply:
wilco